README.md

```diff
@@ -17,18 +17,6 @@
 $ pip install opensquirrel
 ```
 
-### Editable installation
-
-To perform an editable install, run the following command in the root directory of `OpenSquirrel` with the `-e` flag
-```shell
-$ pip install -e .
-```
-
-To install the developer specific dependencies, run the command
```

Comment:
I would end the phrase with a `:`.

Reply:
I've found it.

You have a [commit](/IdeaProjects/OpenSquirrel) from 15/12/2023 9:46, `Add documentation of use of Dev Container in PyCharm`, where you removed that paragraph: `Editable installation`.

If it was an error, you can put it back.